packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -27,6 +27,57 @@ export 'package:flutter/rendering.dart' show
 /// of the existing subclasses that provide adaptors to builder callbacks or
 /// explicit child lists.
 ///
+/// {@template flutter.widgets.sliverChildDelegate.lifecycle}
```

Comment:
is this block exactly the same as the stuff above? if so, can we use the macro there too?

Reply:
Not quite, the ListView one doesn't reference slivers and child delegates and stuff to make it easier to understand without adding more concepts to list view itself. I wish there was a way to do {@macro foo(param1, param2)} etc.